src/component/material-ui/nav-wrapper/index.js

```diff
@@ -12,12 +14,17 @@ class NavWrapper extends Component {
       anchorEl: null,
       dialogOpen: true,
       popoverOpen: false,
+      viewingSaved: false,
     }
 
   handleToggle = () => {
```

Comment:
we should probably make this more specific since there are multiple toggles now.

Reply:
removed, not needed, forgot to pull it out, same as above